submissions/MarharytaBoichenko/dom_menu/js/index.js

```diff
@@ -0,0 +1,38 @@
+import { navList, container } from "./refs.js";
+import quotations from "../data/quotations.json" assert { type: "json" };
+
+const createNavigation = () => {
+  const buttons = [];
+  for (let i = 0; i < 10; i++) {
+    const navButton = ` <li class="nav__item">
+                    <button type="button" class="nav__button" data-number=${i + 1}>${
+      i + 1
+    }</button>
+                </li>`;
+    buttons.push(navButton);
+  }
+
+  navList.insertAdjacentHTML("afterbegin", buttons.join(""));
+};
+createNavigation();
```

Comment:
Since you run this code right away and only once, is there a reason to wrap it in a function?